test/lit.cfg

```diff
@@ -425,7 +425,7 @@ if platform.system() == 'Darwin' and (run_os == 'macosx' or run_os == 'darwin'):
     # <rdar://problem/16678410> Adding more libraries with -lfoo to REPL is broken
     config.available_features.add('swift_repl')
     config.available_features.add('swift_interpreter')
-elif platform.system() == 'Linux':
+elif platform.system() == 'Linux' and not (platform.processor().startswith('armv6') or platform.processor().startswith('armv7')):
```

Comment:
@jckarter As of now, RuntimeDyld complains about unsupported relocations, but it's also the reason why -Bsymbolic is needed in the first place for ELF/ARM.